core-services/image-mirroring/openshift/mapping_origin_4_4

```diff
@@ -30,6 +30,7 @@ registry.svc.ci.openshift.org/origin/4.4:cluster-image-registry-operator quay.io
 registry.svc.ci.openshift.org/origin/4.4:cluster-ingress-operator quay.io/openshift/origin-cluster-ingress-operator:4.4 quay.io/openshift/origin-cluster-ingress-operator:4.4.0 quay.io/openshift/origin-cluster-ingress-operator:latest
 registry.svc.ci.openshift.org/origin/4.4:cluster-kube-apiserver-operator quay.io/openshift/origin-cluster-kube-apiserver-operator:4.4 quay.io/openshift/origin-cluster-kube-apiserver-operator:4.4.0 quay.io/openshift/origin-cluster-kube-apiserver-operator:latest
 registry.svc.ci.openshift.org/origin/4.4:cluster-kube-controller-manager-operator quay.io/openshift/origin-cluster-kube-controller-manager-operator:4.4 quay.io/openshift/origin-cluster-kube-controller-manager-operator:4.4.0 quay.io/openshift/origin-cluster-kube-controller-manager-operator:latest
+registry.svc.ci.openshift.org/origin/4.4:cluster-kube-descheduler-operator quay.io/openshift/origin-cluster-kube-descheduler-operator:4.4 quay.io/openshift/origin-cluster-kube-descheduler-operator:4.4.0 quay.io/openshift/origin-cluster-kube-descheduler-operator:latest
```

Comment:
@smarterclayton we will proceed with this, but I'd like to hear from you if this is ok change, since you own this bit of code.